glusterd2/commands/volumes/volume-smartvol-txn.go

```diff
@@ -1,22 +1,19 @@
-package smartvol
+package volumecommands
 
 import (
 	"os"
-	"path"
 
 	"github.com/gluster/glusterd2/glusterd2/gdctx"
 	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/pkg/api"
 	"github.com/gluster/glusterd2/plugins/device/deviceutils"
-	smartvolapi "github.com/gluster/glusterd2/plugins/smartvol/api"
-
-	config "github.com/spf13/viper"
 )
 
 func txnPrepareBricks(c transaction.TxnCtx) error {
-	var req smartvolapi.Volume
-	if err := c.Get("reqBricksPrepare", &req); err != nil {
+	var req api.VolCreateReq
+	if err := c.Get("req", &req); err != nil {
 		c.Logger().WithError(err).WithField(
-			"key", "reqBricksPrepare").Debug("Failed to get key from store")
+			"key", "req").Debug("Failed to get key from store")
```

Comment:
question:
This should be Debug or Error message

Reply:
not changed this part of the code while moving from plugins/smartvol. Will update this as required